### ISSUE TITLE
OpenSource Community info required

### ISSUE THREAD

**Issue by PulkitSinghDev** [PulkitSinghDev/OpenSource-for-Everyone#7]
Hey 👋🏼 ,
Please take a look at this folder: "[OpenSource_Communities](https://github.com/PulkitSinghDev/OpenSource-for-Everyone/tree/main/OpenSource_Communities)"
Currently, we need more contributions in this `"README.md"`  for releasing our new version of this series 
Do let me know if anyone would like to help us with it :)

**Comment by Panquesito7** [comment]
What about adding [The Algorithms](https://the-algorithms.com) to the list? 🙂

**Comment by PulkitSinghDev** [comment]
we require more communities to be added, so I am just reopening this issue :)

**Comment by hoopinwhoopin** [comment]
Could i add my community

**Comment by PulkitSinghDev** [comment]
If that particular community matches with the theme, then you can add it  @hoopinwhoopin

**Comment by aniruddha-5** [comment]
How about adding google developer students club to this?

**Comment by PulkitSinghDev** [comment]
sure @aniruddha-5 , you can if they are doing some active opensource stuffs

**Comment by aniruddha-5** [comment]
@PulkitSinghDev yes yes they are hyper active and regular sessions are conducted related to open source in discord as well as you tube. I am making a pull request then, you can check it out if it can be included in the list.
